client/me/account/main.jsx

```diff
@@ -884,6 +853,7 @@ const Account = createReactClass( {
 				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
 				<FormattedHeader brandFont headerText={ translate( 'Account Settings' ) } align="left" />
 
+				<SectionHeader label={ translate( 'Account Information' ) } />
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Clear account information' )` *ES Score: 6*

Reply:
Nope these are different.